tests/framework/cli/test_cli_hooks.py

```diff
@@ -97,7 +97,7 @@ def test_kedro_cli_should_invoke_cli_hooks_from_plugin(
     ):
         caplog.set_level(logging.DEBUG, logger="kedro")
 
-        Module = namedtuple("Module", ["cli"])
+        # Module = namedtuple("Module", ["cli"])
```

Comment:
Same here re: comments and the mocker part below.

Reply:
removed!